docs/Cloud_integrations/Device_Integration_Center/webhooks.md

```diff
@@ -47,7 +43,7 @@ Provide **basic information** such as the name and description, optionally enabl
 
     **Telemetry events** include data which is generated by the device. Events can originate from Objects, Object Instance, Resources or Resource Instances.
```

Comment:
Is there a reason behind capitalizing the following: Objects, Object Instance, Resources or Resource Instances?

Reply:
Nope, it can be lowercase